carbon-relay-ng/parameters/os/Fedora.yaml

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
```

Comment:
There are many empty files included here, like this one, which serve no real purpose.  Unless these placeholders are absolutely necessary, I'd recommend removing all of them (under the `os`, `os_family`, `osarch` & `osfinger` subdirectories).